<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="snapshotButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Snapshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recordButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>490</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>490</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
